README.md

```diff
@@ -19,5 +19,22 @@ You can use the provided macro in lib.sls to generate the required configuration
 ### Usage with Zend Server module
 The lookup part (commented in pillar.example) is **required** when using non-default PHP stacks such as Zend Server.
 
+## PHP-FPM Versions
+
+The phpfpm module recognizes either the vanilla PHP-FPM 5.5 installation, or any combination of PHP-FPM 5.6 and PHP-FPM 7, provided by Ondřej Surý via the PPA.
```

Comment:
Mja heb uberhaupt geen idee of we dan nog zut moeten aanpassen, ik ben bang dat het out of the box toch al niet werkt... vandaar dat ik me nu enkel even op 14.04 richtte.